Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+fossaCred = [
+    $class: 'StringBinding',
+    credentialsId: 'fossa-api-key',
+    variable: 'FOSSA_API_KEY'
+]
+
+hubCred = [
+    $class: 'UsernamePasswordMultiBinding',
+    usernameVariable: 'REGISTRY_USERNAME',
+    passwordVariable: 'REGISTRY_PASSWORD',
+    // credentialsId: 'coreeng-hub.docker.com',
+    credentialsId: 'orcaeng-hub.docker.com',
+]
+
+githubCred = [
+    $class: 'StringBinding',
+    variable: 'GITHUB_AUTH_TOKEN',
+    credentialsId: '3b0aa268-7549-4a0f-8cf4-8d75383bd05f', // 5K API rate limit
+]
+
+@Library('jps')
+_
+
+pipeline{
+  agent {
+    node {
+      label 'ubuntu-1804-overlay2'
+    }
+  }
+  triggers {
+    pollSCM('H * * * *')
```

Comment:
Do you want to run this on a cron?

Reply:
I've got the impression that's the only way to do it with declarative?